Replace hardcoded links with refs and docs (#20839)
digital-asset/daml: sdk/docs/sharable/building-applications.rst

As with the Writing Daml section, you can find the Daml code for the example application and features `here <https://github.com/digital-asset/daml/tree/main/docs/source/daml/intro/daml>`_ or download it using the Daml assistant. For example, to load the sources for section 1 into a folder called ``intro1``, run daml new intro1 --template daml-intro-1.

> To run the examples, you will first need to :doc:`install the Daml SDK <getting-started/installation>`.

Reviewer: ```suggestion
To run the examples, you must first :doc:`install the Daml SDK <getting-started/installation>`.
```
